docs/css/bnomics-input.css

```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css?family=IBM+Plex+Mono');
```

Comment:
I thought the idea was to make the component usable with just script inclusion, which shall include styling. I'll separate the Core Styling from Customised Styling to prevent this.

But in a broader note, we don't change the styling of elements in any way, maybe some minor ones, but nothing major.

Reply:
The idea is to allow a user to add the copy functionality easily to an existing HTML element. This should work for different website designs to allow the copy component to be used by a wider range of use cases.